content/400-reference/100-system-requirements.mdx

```diff
@@ -21,7 +33,48 @@ Notes:
 
 See also: [Supported database versions](database-reference/supported-databases)
 
-</TopBlock>
+### Operating systems
+
+Prisma is supported on MacOS, Windows and most Linux distributions.
+
+#### Linux runtime dependencies
+
+Prisma requires the following system libraries installed to work:
+
+- OpenSSL 1.0.x (except Alpine Linux), 1.1.x or 3.x
+- zlib (`libz.so.1`)
+- libgcc (`libgcc_s.so.1`)
+- C standard library (glibc on most Linux distributions or musl libc on Alpine Linux)
+
+The following two tables show the supported Linux distro' family, OpenSSL versions and C standard libraries for each CPU architecture.
+Note that, in case of Prisma not being able to resolve the OpenSSL version on your system (e.g. because it is not installed), it will default to OpenSSL 1.1.x
+
+On `AMD64` (`x86_64`) architectures:
+
+| Distro family    | OpenSSL version   | libc version | 
+| ---------------- | ----------------- | ------------ |
+| Alpine           | 1.1.x, 3.x        | musl 1.2.x+  |
+| RHEL             | 1.0.x, 1.1.x, 3.x | glibc 2.17+  |
+| Debian or others | 1.0.x             | glibc 2.19+  |
+| Debian or others | 1.1.x, 3.x        | glibc 2.24+  |
+
+On `ARM64` (`aarch64`) architectures:
+
+| Distro family    | OpenSSL version   | libc version | 
+| ---------------- | ----------------- | ------------ |
+| Alpine           | 1.1.x, 3.x        | musl 1.2.x+  |
```

Comment:
```suggestion
| Alpine           | 1.1.x, 3.x        | musl 1.2.x   |
```

We don't know yet if we are compatible with musl 1.3 before 1.3 exists, it is not necessarily backward compatible in the second digit. Musl 1.2 had a breaking change for 32-bit architectures, for example. On all architectures we officially compile for we are actually compatible with 1.1 as well as far as I tested, but better to document only 1.2.x since that's what we compile against.